index.bs

```diff
@@ -736,6 +736,16 @@ Each {{XRSpace}} has a <dfn for="XRSpace">native origin</dfn> that is tracked by
 
 The [=effective origin=] of an {{XRSpace}} can only be observed in the coordinate system of another {{XRSpace}} as an {{XRPose}}, returned by an {{XRFrame}}'s {{XRFrame/getPose()}} method. The spatial relationship between {{XRSpace}}s MAY change between {{XRFrame}}s.
 
+Each {{XRSpace}} has an <dfn for="XRSpace">internal type</dfn>, used to internally distinguish the various kinds of spaces. This value can be <dfn for="XRSpace">viewer space</dfn>, <dfn for="XRSpace">input target ray</dfn>, <dfn for="XRSpace">input grip</dfn>, or one of the variants of {{XRReferenceSpaceType}}.
+
+An {{XRSpace}} of type [=XRSpace/input target ray=], [=XRSpace/input grip=] must have an associated {{XRInputSource}} <dfn for="XRSpace">input source</dfn>.
+
+The [=native origin=] of an {{XRSpace}} with [=XRSpace/internal type=] of [=XRSpace/viewer space=] is the position and orientation of the [=viewer=].
```

Comment:
Should we perhaps use the term "physical position and orientation" or "physical pose" consistently throughout for these definitions, to emphasize when a WebXR concept is being defined in terms of physical reality, rather than the logical position and orientation of another WebXR construct within some space?

Reply:
I'm ambivalent about this, happy to add it if people agree.

Reply:
I don't object to it, and I do feel like it helps add clarity.